src/test/utils/testUtils.ts

```diff
@@ -164,7 +171,63 @@ export function getMvnProjectPath(): string {
     return foundText;
   }
 
-  
+export async function isViewSectionEmpty(section: ViewSection): Promise<boolean> {
+    try {
+        const items = await section.getVisibleItems();
+        console.log("items...."+(items.length === 0));
+        return items.length === 0;
+    } catch (error) {
+      // if there is no visible elements, getVisibleItems() method is throwing timeout error.
+        return true;
+    }
+}
+
+export async function findFileRecursively(name: string,  parentItem?: TreeItem): Promise<TreeItem | undefined> {
+  let explorerSection;
+  if (!parentItem) {
+    explorerSection = await new SideBarView().getContent().getSection('liberty.maven.test.wrapper.app');
+}
+
+  const items = parentItem 
+      ? await parentItem.getChildren() 
+      : await explorerSection?.getVisibleItems();
+
+  if (!items) return undefined;
+
+  for (let item1 of items) {
+    let item = item1 as TreeItem;
+      if (await item.getLabel() === name) {
+          return item; // File found
+      }
```

Comment:
recommended change for better naming practices:
```suggestion
  for (let item of items) {
    let treeItem = item as TreeItem;
      if (await treeItem.getLabel() === name) {
          return treeItem; // File found
      }
```